services/tables/src/main/java/com/linkedin/openhouse/tables/common/DefaultReplicationInterval.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.openhouse.tables.common;
+
+/**
+ * ENUM for default replication interval associated with Interval in {@link
+ * com.linkedin.openhouse.tables.api.spec.v0.request.components.ReplicationConfig}
+ */
+public enum DefaultReplicationInterval {
```

Comment:
Should avoid naming the class with Classifier `Default`. 
Maybe this structure:
```java
enum ReplicationInterval {
     DEFAULT("24H");
}
``` 
Leaves scope to add other intervals too.